src/concourse.rs

```diff
@@ -20,6 +21,7 @@ pub(crate) struct Build {
     // job_name: String,
     // api_url: String,
     // pipeline_name: String,
+    // pipeline_instance_vars: Option<Map<String, Value>>,
```

Comment:
Question: does `serde_json` actually use this in some magical way, or is it just there for completeness?